packages/sql/package.json

```diff
@@ -10,8 +10,9 @@
   "license": "BSD-3-Clause",
   "author": "Jeffrey Heer (http://idl.cs.washington.edu)",
   "type": "module",
-  "main": "src/index.js",
-  "module": "src/index.js",
+  "types": "dist/mosaic-sql.d.ts",
+  "main": "dist/mosaic-sql.js",
+  "module": "src/index.ts",
```

Comment:
Is this ok? I thought we would need to always point to js files here.

Reply:
You're right, will address.